_announcements/01_fpki-repository-migration.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+navtitle: Upcoming Migration of Federal PKI Certificate Repository Services
+title: Upcoming Migration of Federal PKI Certificate Repository Services
+pubDate: April 1, 2019
+collection: announcements
+permalink: announcements/fpki-repository-migration/
+description: On April 22, 2019, the Federal Public Key Infrastructure Management Authority will migrate the hosting of HyperText Transfer Protocol (HTTP) repository services to a cloud-based solution. This announcement provides additional information related to the upcoming migration.
+---
+
+{% include alert-info.html content="Upcoming changes to the hosting of Federal Public Key Infrastructure Certification Authority (CA) data repositories could impact your agency." %}
+
+On April 22, 2019, the Federal Public Key Infrastructure Management Authority will migrate the hosting of HyperText Transfer Protocol (HTTP) repository services to a cloud-based solution. Existing Federal PKI CA certificate Uniform Resource Locators (URLs) **will not** change as a result of this migration.
+
+## What will be impacted?
+
+This change will affect the hosting of certificate revocation lists, CA certificates, and certificate bundles for the following Federal PKI CAs:
+- Federal Bridge CA 2016
+- Federal Common Policy CA
+- SHA1 Federal Root CA
+- CAs operating within the FPKI Community Interoperability Test Environment (CITE)
+- US Federal TLS Root CA (under development) 
```

Comment:
```suggestion
```

Reply:
If it's the Dev / Test CA, let's be specific. 
The production CA doesn't exist yet so we don't need to specify this in the announcement (reduce confusion).